netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -80,15 +169,15 @@ static net_ebpf_extension_hook_provider_t*
 //
 
 static ebpf_result_t
-net_ebpf_extension_sock_addr_on_client_attach(
+_net_ebpf_extension_sock_addr_on_client_attach(
     _In_ const net_ebpf_extension_hook_client_t* attaching_client,
     _In_ const net_ebpf_extension_hook_provider_t* provider_context)
 {
     ebpf_result_t result = EBPF_SUCCESS;
     const ebpf_extension_data_t* client_data = net_ebpf_extension_hook_client_get_client_data(attaching_client);
     uint32_t compartment_id;
     uint32_t wild_card_compartment_id = UNSPECIFIED_COMPARTMENT_ID;
-    net_ebpf_extension_wfp_filter_parameters_t* filter_parameters = NULL;
+    net_ebpf_extension_wfp_filter_parameters_array_t* filter_parameters = NULL;
```

Comment:
maybe rename parameter to match type?
```suggestion
    net_ebpf_extension_wfp_filter_parameters_array_t* filter_parameters_array = NULL;
```

Reply:
Done.